go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/getkin/kin-openapi v0.124.0
 	github.com/google/go-cmp v0.6.0
 	github.com/kong/kubernetes-ingress-controller/v2 v2.12.3
+	github.com/olekukonko/tablewriter v0.0.5
```

Comment:
@robscott I found that this is allowlisted, can you double check I am right?

Reply:
Yep, looks like this is [very widely used in k8s repos](https://cs.k8s.io/?q=github.com%2Folekukonko%2Ftablewriter&i=nope&files=&excludeFiles=&repos=), thanks for checking!